src/presentation/controller/postTransition/routing/limitedPartnership.ts

```diff
@@ -116,6 +117,18 @@ const postTransitionRoutingTermChangeCheckYourAnswers = {
   }
 };
 
+const PRINCIPAL_PLACE_OF_BUSINESS_ADDRESS_KEY = "principalPlaceOfBusinessAddress";
+
+const postTransitionRoutingPrincipalPlaceOfBusinessAddress = {
+  previousUrl: url.LANDING_PAGE_URL,
+  currentUrl: url.ENTER_PRINCIPAL_PLACE_OF_BUSINESS_ADDRESS_URL,
+  nextUrl: url.WHEN_DID_THE_PRINCIPAL_PLACE_OF_BUSINESS_CHANGE_URL,
+  pageType: PostTransitionPageType.enterPrincipalPlaceOfBusinessAddress,
+  data: {
+    titleKey: PRINCIPAL_PLACE_OF_BUSINESS_ADDRESS_KEY
```

Comment:
is the titleKey needed? other usages of it are for the 'when did x change' screens and used to vary the page title on the shared page.

Reply:
That's a fair point,  we won't need it for this page so removed for this PR.

Reply:
i can still see it in the PR and there haven't been any new commits so won't re-review yet